var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -86,6 +87,14 @@ class Llvm(CMakePackage, CudaPackage):
         default=True,
         description="Build the LLVM C++ standard library",
     )
+    variant(
+        "clang_cxx_stdlib",
+        default="default",
+        description="C++ standard library used by clang++. "
+        "Default is libc++ when +libcxx or libstdc++ otherwise. "
+        "The selection is effective only if +libcxx is specified",
+        values=("default", "libstdc++", "libc++"),
```

Comment:
Not sure if spack check input values against reference values at the first place.